src/esp/core/CMakeLists.txt

```
@@ -1,5 +1,21 @@
 file(GLOB_RECURSE core_SOURCES "*.cpp")
 
+if(NOT BUILD_GUI_VIEWERS AND UNIX AND NOT APPLE)
```

```suggestion
if(NOT BUILD_GUI_VIEWERS AND CORRADE_TARGET_UNIX AND NOT CORRADE_TARGET_APPLE)
```

... because the builtin `UNIX` is unfortunately defined for Emscripten as well, which is probably not what you want. The `CORRADE_TARGET_UNIX` explicitly isn't.

(Sorry for pushing for these all the time :sweat_smile:)